chartfx-math/src/main/java/de/gsi/math/DataSetMath.java

```diff
@@ -95,7 +94,7 @@ public static DataSet averageDataSetsFIR(final List<DataSet> dataSets, final int
             double eyp = 0.0;
 
             int count = 0;
-            for (int j = Math.max(0, dataSets.size() - nAvg); j < dataSets.size(); j++) {
+            for (int j = java.lang.Math.max(0, dataSets.size() - nAvg); j < dataSets.size(); j++) {
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'java.lang.Math.max' due to existing implicit import 'java.lang.*'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)